ksql-cli/src/main/java/io/confluent/ksql/cli/console/Console.java

```diff
@@ -110,7 +166,7 @@ public Console(final OutputFormat outputFormat, final KsqlRestClient restClient)
 
   protected abstract LineReader buildLineReader();
 
-  protected abstract void puts(InfoCmp.Capability capability);
+  public abstract void puts(InfoCmp.Capability capability);
```

Comment:
This is only used in one place AFAICT. Maybe we should just make it `clear()` and then not expose the `JLine` specific enum?

Reply:
Makes sense. (Was trying to minimise changes....)